### PR TITLE
Added recipe for `pytorch_toolbelt`

### PR DESCRIPTION
**Pull request by sarthakpati**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by sarthakpati** [comment]
@conda-forge/help-python this package does not use `c_stdlib_version` for build, and the recipe does not have `MACOSX_DEPLOYMENT_TARGET` either. Could you perhaps guide me on what needs to be done to fix the linter?

**Comment by synapticarbors** [comment]
This looks like it shouldn't have any compiled code, so you should be able to remove the `conda_build_config.yml` and the `build` section under `requirements`.

**Comment by synapticarbors** [comment]
Also note that the windows build will always fail since we don't have pytorch available on conda-forge. This is ok and can be ignored.

**Comment by sarthakpati** [comment]
> This looks like it shouldn't have any compiled code, so you should be able to remove the `conda_build_config.yml` and the `build` section under `requirements`.

I believe there is an underlying dependency that needs the C compiler, because of which I was getting that linter error before.

File: recipes/pytorch_toolbelt/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/pytorch_toolbelt-{{ version }}.tar.gz
+  sha256: a7a5126fbaeea68d7f7423cef540051554159f6f57b3af07dba8ee5b166cc9b0
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
```
Comment (synapticarbors) [inline recipes/pytorch_toolbelt/meta.yaml:19]: You should also remove this `build` section.

**Comment by sarthakpati** [comment]
This is so weird. The [latest commit](https://github.com/conda-forge/staged-recipes/blob/057ea8bf9f0436ad68b7d9d17a28867d7a4bf1ba/recipes/pytorch_toolbelt/meta.yaml) and [the commit where I had the lint issue](https://github.com/conda-forge/staged-recipes/blob/758c490fd632c95a80962aa4b2037dfa29678d66/recipes/pytorch_toolbelt/meta.yaml) are basically the same. But the former did not give any linter issue. Any idea why?

**Pull request merged**